src/main/java/minicraft/screen/OptionsMainMenuDisplay.java

```diff
@@ -0,0 +1,30 @@
+package minicraft.screen;

```

Comment:
Why does OptionsDisplay still exist?

Reply:
I said in #205, that I kept it just in case.